tls/tests/conftest.py

```diff
@@ -0,0 +1,182 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from datadog_checks.tls.utils import days_to_seconds
+
+from .utils import download_cert, temp_binary
+
+
+@pytest.fixture(scope='session', autouse=True)
```

Comment:
Could we use an non corporate domain like https://example.com/ instead of google.com?

Reply:
I'd prefer something that's always up so tests aren't flaky